src/components/LeftSidebar/ConversationsList/ConversationsList.vue

```diff
@@ -70,18 +70,25 @@ export default {
 			this.fetchConversations()
 		}, 30000)
 
-		EventBus.$on('routeChange', ({ from, to }) => {
+		// Save function to be able to remove it when set as an EventBus
+		// handler.
+		this.routeChangeHandler = ({ from, to }) => {
```

Comment:
instead of defining a method inside this function, put it in the `methods` object. That is exactly why it exists?